assemblies/samples/src/assembly/assembly.xml

```diff
@@ -26,6 +26,7 @@
         <exclude>/src/</exclude>
         <exclude>/target/</exclude>
       </excludes>
+      <fileMode>0755</fileMode>
```

Comment:
This is going to give execution permissions to all files, not just the shell scripts. Is this intentional? Comparing to the files from the 7.0 artifact, only the '.sh' files have the execution permissions.